src/shapes/Object/InteractiveObject.ts

```diff
@@ -414,9 +414,8 @@ export class InteractiveFabricObject<
     ctx.save();
     ctx.translate(options.translateX, options.translateY);
     ctx.lineWidth = 1 * this.borderScaleFactor;
-    if (!this.group) {
-      ctx.globalAlpha = this.isMoving ? this.borderOpacityWhenMoving : 1;
-    }
+    ctx.globalAlpha = this.isMoving ? this.borderOpacityWhenMoving : 1;
```

Comment:
If the fix is just this, i need to test the combinations of feature that will pass here.